libs/minecraft/PosUtils.arucas

```diff
@@ -136,4 +175,249 @@ enum Direction {
         }
         return Direction.NONE;
     }
+    static fun values(){
```

Comment:
This is overwriting the default `values` method. This should not be `values` instead rename it, e.g. `getAllDirections`